<launch>

    <!-- Robot namespace -->
    <arg name="robot_name" default="my_gen3"/>
    <arg name="start_delay_seconds" default="1"/>

    <!-- Launch the example in that namespace -->
    <node name="depth_colour" pkg="arm" type="depth.py" ns="$(arg robot_name)" output="screen" launch-prefix="bash -c 'sleep $(arg start_delay_seconds); $0 $@' "/>

</launch>